lib/xcodebuild.js

```diff
@@ -215,6 +215,11 @@ export class XcodeBuild {
     /** @type {string[]} */
     const args = [];
 
+    if (this.useSimpleBuildTest) {
```

Comment:
I would rather show this warning in the xcuitest driver as we are going to remove the cap from there first

Reply:
Sure > xcuitest driver

> major version

Yea. I was wondering if we could add more deprecations which may be no longer needed and remove them at the same time with major version increment.